### PR TITLE
re-factored card component

### PR DESCRIPTION
**Pull request by Manju-sree**
re-factored card component

### CODE REVIEW

File: Components/Card/card.css [hunk 1]
```diff
@@ -86,7 +90,7 @@
         height: 1.2rem;
     }
     
-    .fa-heart {
+    .heart-pink {
         color: #fc547b;
     }
```
Comment (sushil-mod) [inline Components/Card/card.css:95]: color variable

**Review by sushil-mod — APPROVED** [review]
just one comment , rest is good

**Review by ShreyashMahajan — APPROVED** [review]
good to merge

File: Components/Card/card.css (hunk 1 above)
Comment (Manju-sree) [reply Components/Card/card.css:95]: done changes..

**Pull request merged**